### DISCUSSION TITLE
Animation when moving windows

### DISCUSSION THREAD

**Discussion by florentinf** [mrkai77/Loop#446] [Q&A] [answered]
Hello everyone.
I am a new user of Loop and have a problem with the app:
When I hold down the trigger button (ctrl), moving and resizing the app Loop works without any problems. But when I use any other app, it doesn't move animatedly, it just moves to another position without animation.
Does anyone have an idea why this only works with the Loop app and nowhere else?
Thanks for an answer!

https://github.com/MrKai77/Loop/assets/166118119/ff9159c1-51a0-4781-926e-3873364751cd

**Comment by mrkai77** [comment, marked as answer]
Oh, that seems like a bug. If you want to enable window animations for other apps, go to settings -> advanced and enable "Animate window resize"

**Reply by florentinf** [reply]
Thank you very much for your answer! I activated this function and Loop crashed after changing the size twice. Attached is a screen recording.

https://github.com/MrKai77/Loop/assets/166118119/80dae289-005f-436b-9e5b-85cbc7f53923

**Reply by mrkai77** [reply]
Yes, that is why it is currently marked as a alpha feature :)

Just wondering, can you reproduce this when your animation speed is set to fast, or in any other apps?

**Reply by florentinf** [reply]
I tried it and found out:
1. the error occurs in all apps
2. it only occurs if you use Loop more than three times in one second, which admittedly you don't do very often :)

I have another function suggestion: 
If I set the app to open automatically when the Mac boots up, the app is launched with the settings every time. it would be great if you could activate that the app starts in the background and you can just use the app without always having to minimise the Loop app first.
Do you understand?
